optee-utee/src/uuid.rs

```diff
@@ -79,7 +79,12 @@ impl Uuid {
         ))
     }
 
-    /// Crates a raw TEE client uuid object with specified parameters.
+    /// Creates a `Uuid` from a raw `TEE_UUID` object.
+    // pub fn from_raw(raw: raw::TEE_UUID) -> Uuid {
+    //     Self { raw }
+    // }
```

Comment:
All above comments are resolved. BTW forgot to delete these lines, will delete in next commit for adding review tags.